src/kvaser_interface.cpp

```diff
@@ -155,13 +129,15 @@ ReturnStatuses KvaserCan::close()
     return ReturnStatuses::CHANNEL_CLOSED;
 
   // Close the channel
-  if (canClose(*handle) != canOK)
-    return ReturnStatuses::CLOSE_FAILED;
+  canStatus ret = canClose(*handle);
 
   *handle = -1;
   on_bus = false;
 
-  return ReturnStatuses::OK;
+  if (ret != canOK)
+    return ReturnStatuses::CLOSE_FAILED;
```

Comment:
Always use curly braces

Reply:
I'll fix formatting by a quick application of `clang-format` in another PR. Applying that change globally would cause this PR to be pretty hard to read.

Reply:
I was just referring to these couple lines, but I'll assign you a separate issue to format the code.